apps/journeys-admin/src/libs/firebaseClient/UseFirebase.tsx

```diff
@@ -0,0 +1,34 @@
+import { getAuth, signOut } from 'firebase/auth'
+import { firebaseClient } from './firebaseClient'
+import { useRouter } from 'next/router'
+import { useAuthState } from 'react-firebase-hooks/auth'
+
+interface UseFirebaseProps {
+  logOut: () => void
+  user
+  loading
+  error
+}
+
+export const UseFirebase = (): UseFirebaseProps => {
```

Comment:
only components should start with a capital letter. Also Props refers to the input props provided to a component. This should be renamed to something like UseFirebaseResponse.

```suggestion
export const useFirebase = (): UseFirebaseResponse => {
```